rewrite-gradle/src/main/java/org/openrewrite/gradle/ChangeDependency.java

```diff
@@ -208,10 +235,9 @@ private J.MethodInvocation updateDependency(J.MethodInvocation m, ExecutionConte
                             }
                             if (original != updated) {
                                 String replacement = updated.toStringNotation();
-                                m = m.withArguments(ListUtils.mapFirst(depArgs, arg -> {
-                                    G.GString gString = (G.GString) arg;
-                                    return gString.withStrings(ListUtils.mapFirst(gString.getStrings(), l -> ((J.Literal) l).withValue(replacement).withValueSource(replacement)));
-                                }));
+                                J.Literal newLiteral = literal.withValue(replacement)
```

Comment:
We could employ the `ChangeStringLiteral` helper class here.

Reply:
I considered it, but ChangeStringLiteral was not working fine with GString, since it was taking the delimiters of the literal inside the GString, which is none